src/components/Router/index.js

```diff
@@ -31,15 +30,16 @@ class Router extends Component {
   }
 
   render = (props) => {
-    const { websocketErrorEncountered, options, ...otherProps} = props
+    const { options, ...otherProps} = props
     const defaultStepOptions = ['welcome','document','face','complete']
     const stepOptions = options.steps || defaultStepOptions
     otherProps.nextStep = this.nextStep
+    otherProps.token = options.token
+    otherProps.serverUrl = options.serverUrl
```

Comment:
well. one way to simplify it would be to pass a `postToServer` function with the `token` and `serveurl` binded to it already.